pages/index.vue

```diff
@@ -2295,9 +2264,8 @@ export default {
       }));
     },
     pathInputHandler() {
-      let queryString = this.getQueryStringFromPath(),
-        params = this.queryStringToArray(queryString);
-
+      let queryString = this.getQueryStringFromPath();
```

Comment:
```suggestion
      const queryString = this.getQueryStringFromPath();
```

Reply:
`queryString`'s scope is only required inside that block, so let's go wil `let`